client/src/uiKit/Button/Button.js

```diff
@@ -1,24 +1,76 @@
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import './Button.css';
 
-function Button({ text, variant, onClick }) {
+const Button = ({
+  text,
+  theme = 'primary',
+  variant = 'default',
+  icon,
+  className,
+  onClick,
+  large,
+  small,
+  left,
```

Comment:
Don't need both left & right prop for icon positioning. It should be one prop `isRightIcon`, which could default to false.